website/docs/r/vpc_v1.html.markdown

```diff
@@ -27,6 +27,15 @@ resource "huaweicloud_vpc_v1" "vpc_v1" {
   cidr = "${var.vpc_cidr}"
 }
 
+resource "huaweicloud_vpc_v1" "vpc_with_tags" {
```

Comment:
the example can not work as some required parameters were missing.
please see: https://www.terraform.io/docs/providers/huaweicloud/r/vpc_subnet_v1.html